tests/functional/object/rgw/test_obc_quota.py

```
@@ -191,3 +642,76 @@ def test_obc_quota_full_alert(
+
+        # Fill the bucket with ~90% of maxSize (9 x 1MB objects for 10MB quota)
+        max_size_mb = int(quota["maxSize"].rstrip("M"))
+        fill_amount = (max_size_mb * 90) // 100
+        write_random_test_objects_to_bucket(
+            awscli_pod_session,
+            bucket_name,
+            test_directory_setup.origin_dir,
+            amount=fill_amount,
+            mcg_obj=OBC(bucket_name),
+        )
+        logger.info(f"Filled bucket {bucket_name} with ~90% maxSize capacity")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)